### ISSUE TITLE
I wanted ask 

### ISSUE THREAD

**Issue by Rafci** [eritislami/evobot#1548]
I have a question why when I want to play music, it enters the voice channel and does not play music and disconnects and sends a message saying Could not join the channel: {error} and there is no error on the console?
​

**Comment by jundeUC** [comment]
Its a youtube side problem since they constantly change code.  and also the discord.py needs to be updated.

**Comment by AndrewsPanda** [comment]
Does that mean this project isn't functional at the moment?

**Comment by rockrevenchy** [comment]
Honestly I'm not sure what they mean, the bot is running just fine on my end

**Comment by AndrewsPanda** [comment]
Do you have a fork of the version that works for you?

**Comment by rockrevenchy** [comment]
Nope, straight up updated the bot by pulling content from this repository, little to no modifications from me... Perhaps some dependencies are not available anymore? I'd have to try it out and see

**Comment by luckiesworld** [comment]
I mean like its working rn.
